### PR TITLE
DTSPB-1358: validate copies numbers non-negative

### PR DESCRIPTION
**Pull request by sanjayparekh-hmcts**
This PR fixes the issue for https://tools.hmcts.net/jira/browse/DTSPB-1358

Additionally small change for https://tools.hmcts.net/jira/browse/DTSPB-36. Details are below 

Fix the order of Upload Doc types to be as per DTSPB-36 ie:

Will
Email
Correspondence 
Codicil
Death certificate
Other

### CODE REVIEW

**Review by isha2040 — APPROVED** [review]

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
...
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "extraCopiesOfGrant", "Label": "How many extra UK copies of the grant do you need?", "FieldType": "Number", "Min": "0", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "outsideUKGrantCopies", "Label": "How many sealed and certified copies of the grant do you need for use outside the UK?", "HintText": "Certain non-UK territories may need your client to provide a sealed and certified grant. Check if this is needed for your case, as a UK grant is still valid in some non-UK countries and jurisdictions", "FieldType": "Number", "Min": "0", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "applicationFee", "Label": "Application fee", "FieldType": "MoneyGBP", "SecurityClassification": "Public"},
```
Comment (vinceholt) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json:48]: Very minor - would be a bit better if the Min0 property was in the same place within the json properties for both entries

**Review by vinceholt — APPROVED** [review]
I've approved but left one 'COULD' comment that could be attended if you wished

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json (hunk 1 above)
Comment (sanjayparekh-hmcts) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json:48]: Yes - good point - definitely do that

Comment (sanjayparekh-hmcts) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json:48]: HOLD on - it IS ALREADY in the same place, just that the second line has a massive hint text on it - its correct as it is now

**Review by arun-hmcts — APPROVED** [review]

**Pull request closed without merging**